Change notation for mass flow rate from \dot{m} to \tilde{m}. (#2645)
modelica/ModelicaSpecification: chapters/derivationofstream.tex

\section{Reasons for avoiding the actual mixing enthalpy in connector definitions}\label{reasons-for-avoiding-the-actual-mixing-enthalpy-in-connector-definitions}

> Consider a connection set with \emph{n} connectors, and for the mass flow rates use $\tilde{m}=\text{m\_flow}$.

Reviewer: The `m_flow` looks like `\lstinline` stuff to me:
```suggestion
Consider a connection set with $n$ connectors, and denote the mass flow rates \lstinline!m_flow! by $\tilde{m}$.
```

> Consider a connection set with \emph{n} connectors, and for the mass flow rates use $\tilde{m}=\text{m\_flow}$.

Reviewer: Isn't the number of connectors typically denoted `$N$` (upper case) in this chapter?  (Just wondering; I can see that it is outside the scope of the present fix.)

Author: It's unfortunately messier. The formulas use $n$ and the headings use $N$ (or N - I believe there was some issue with math in headings). There might be some exceptions. For now I view that as good enough and rationalize the N as heading-capitalization.

Reviewer: The upper case symbols aren't only in headings: I think it starts here, followed by lots of other places in _stream.tex_:
> For the following definition it is assumed that $N$ inside connectors \lstinline!$m_{j}$.c! ($j = 1, 2, \ldots, N$) and $M$ outside connectors \lstinline!$c_{k}$! ($k = 1, 2, \ldots, M$) belonging to the same connection set

Reviewer: On the other hand, it is ugly to use uppercase `$N$` and `$M$`, so I'd be strongly in favor of going in the direction of `$n$` and `$m$`, but don't know what conflicts that would cause.

Author: Ah, I see in the stream-chapter `N` and `M` are used in the Modelica code, but in this appendix that is mostly math $n$ and $m$ are used in the formulas and N and M in the heading.

I agree that $n$ and $m$ and also `n` and `m` would be more common, but will place that in another PR (it's primarily for the stream-chapter).
